pai-management/doc/cluster-bootup.md

```diff
@@ -5,85 +5,113 @@ This document introduces the detailed procedures to boot up PAI on a cluster. Pl
-
-- A [dev-box](./how-to-setup-dev-box.md) has been set up and can access the cluster.
-- SSH service is enabled on each of the machines.
-- All machines share the same username / password for the SSH service on each of them.
-- The username that can be used to login to each machine should have sudo privilege.
-- All machines to be set up as masters should be in the same network segment.
-- A load balancer is prepared if there are multiple masters to be set up.
+We assume that the whole cluster has already been configured by the system maintainer to meet the [Prerequisites](../../README.md#how-to-deploy).
 
 With the cluster being set up, the steps to bring PAI up on it are as follows:
 
+- Step 0. Prepare dev-box
 - Step 1. Prepare PAI configuration.
     - (For advanced users) This step can either be done by writing the configuration files manually,
     - (For novice users) or be done using the `paictl` tool.
 - Step 2. Boot up Kubernetes.
 - Step 3. Start all PAI services.
 
-## Step 1a. Prepare PAI configuration: Manual approach <a name="step-1a"></a>
+## Quick deploy with default settings <a name="quickdeploy"></a>
+### Step 0. Prepare the dev-box
 
-This method is for advanced users. PAI configuration consists of 4 YAML files:
+It is recommended to perform the operations below in a dev box.
+Please refer to this [section](../pai-management/doc/how-to-setup-dev-box.md) for the details of setting up a dev-box.
 
-- [`cluster-configuration.yaml`](./how-to-write-pai-configuration.md#cluster_configuration) - Machine-level configurations, including login info, machine SKUs, labels of each machine, etc.
-- [`kubernetes-configuration.yaml`](./how-to-write-pai-configuration.md#kubernetes_configuration) - Kubernetes-level configurations, part 1. This file contains basic configurations of Kubernetes, such as the version info, network configurations, etc.
-- [`k8s-role-definition.yaml`](./how-to-write-pai-configuration.md#k8s_role_definition) - Kubernetes-level configurations, part 2. This file contains the mappings of Kubernetes roles and machine labels.
-- [`serivices-configuration.yaml`](./how-to-write-pai-configuration.md#services_configuration) - Service-level configurations. This file contains the definitions of cluster id, docker registry, and those of all individual PAI services.
+### Step 1. Prepare the quick-start.yaml file <a name="step-1a"></a>
 
-There are two ways to prepare the above 4 PAI configuration files. The first one is to prepare them manually. The description of each field in these configuration files can be found in [A Guide For Cluster Configuration](how-to-write-pai-configuration.md).
+An example yaml file is shown below. Note that you should change the IP address of the machine and ssh information accordingly.
 
-If you want to deploy PAI in single box environment, please refer to [Single Box Deployment](single-box-deployment.md) to edit configuration files.
+```yaml
+# quick-start.yaml
+
```

Comment:
BTW, as the quick start doc already merged here, suggest to remove the quick start from git repo also.

Reply:
.md for quickstart has been removed, the yaml is for user config file and need keep